product/consul.go

```diff
@@ -43,6 +44,8 @@ func ConsulSeekers(tmpDir string, since, until time.Time) ([]*s.Seeker, error) {
 		s.NewHTTPer(api, "/v1/namespace"),
 		s.NewHTTPer(api, "/v1/status/leader"),
 		s.NewHTTPer(api, "/v1/status/peers"),
+
+		logs.NewDocker("consul", tmpDir, since, until),
```

Comment:
Will this assume that the running container is named "consul," or will Docker match based on image names?

Reply:
Docker CLI calls the param `CONTAINER` but will happily match the image name too.

Reply:
Nice! I did not know that.